src/components/sbb-expansion-panel-content/sbb-expansion-panel-content.scss

```diff
@@ -0,0 +1,21 @@
+@use '../../global/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-expansion-panel-content-padding-inline-start: var(--sbb-spacing-fixed-6x);
+}
+
+:host([data-icon-space]) {
+  --sbb-expansion-panel-content-padding-inline-start: calc(
+    var(--sbb-spacing-fixed-6x) + var(--sbb-spacing-fixed-9x) + var(--sbb-spacing-fixed-4x)
```

Comment:
not sure what is calculated here. maybe we could inherit some vars from expansion panel? or at least better document the calculation

Reply:
Added a comment in commit a9406aff